src/command-parser/parserConfig.ts

```diff
@@ -199,4 +200,56 @@ export const parserConfig = {
     'key-gen': nodeKeyGenArgs,
     'issue-op-cert': opCertSigningArgs,
   },
+  'catalyst': {
+    'voting-key-registration-metadata': {
+      '--mainnet': {
+        nargs: '?',
+        dest: 'network',
+        const: parseNetwork('MAINNET'),
+        default: parseNetwork('MAINNET'),
+        help: 'NETWORK.',
+      },
+      '--testnet-magic': {
+        nargs: '?',
+        dest: 'network',
+        type: (magic: string) => parseNetwork('TESTNET', magic),
+        help: 'Protocol magic number.',
+      },
+      '--vote-public-key': {
+        required: true,
+        dest: 'votePublicKey',
+        type: (path: string) => parseVotePubFile(path),
+        help: 'Input filepath to vote public key.',
+      },
+      '--reward-address': {
+        required: true,
+        dest: 'rewardAddress',
+        help: 'Address to receive voting rewards.',
+      },
+      '--stake-signing-key': {
+        required: true,
+        dest: 'hwStakeSigningFileData',
+        type: (path: string) => parseHwSigningFile(path),
+        help: 'Input filepath of the hardware wallet stake signing file.',
```

Comment:
Help text could maybe be a bit more descriptive here e.g. `Input filepath of the hardware wallet stake signing file, which will be used to to sign the voting registration.`.

Reply:
Refactored.

Reply:
Refactored